test/integration/scheduler_perf/config/scheduler-config-file.yaml.template

```diff
@@ -0,0 +1,77 @@
+apiVersion: kubescheduler.config.k8s.io/v1beta1
```

Comment:
does this template exist only to make it easier to manually create new KubeSchedulerConfiguration files in this directory?

Reply:
yes, I'd hope to take this template as a reference.

Reply:
I prefer we remove this from the PR if we are not going to use it. Also, it's not a template, it's just a yaml.

Reply:
I agree, if this is not being consumed, then it will drift away.

Reply:
removed.